### PR TITLE
📦 NEW:  AI drug assistant

### PR DESCRIPTION
**Pull request by AIENGINE**
Pipe example: AI Drug Assistant

### CODE REVIEW

**Comment by msaaddev** [comment]
@AIENGINE I assume its a duplicate of the current example with a few changes. Can you please list down the files you have changed in this PR so we can review it quickly? Thank you!

**Comment by AIENGINE** [comment]
- updated chat interface text in opening.tsx, readme.md, and header.tsx

Please let me know if more details are required

File: examples/ai-drug-assistant/components/opening.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+          <div className="mt-4 flex flex-col gap-4 text-sm [&>p]:my-0 [&>p]:py-0">
+            <p>Learn more by checking out:</p>
+            <div className="flex flex-col gap-4 mt-2 text-sm">
+              <Dlink href="https://beta.langbase.com/examples/ai-drug-assistant">
+                <span>1.</span>
+                <span>Fork this AI Drug Assistant Pipe on ⌘ Langbase</span>
+              </Dlink>
+              <Dlink href="https://github.com/LangbaseInc/langbase-examples/tree/main/examples/ai-drug-assistant">
+                <span>2.</span>
+                <span>Use LangUI.dev's open source code compnents</span>
```
Comment (msaaddev) [inline examples/ai-drug-assistant/components/opening.tsx:40]: There is a typo in `components`

File: examples/ai-drug-assistant/package.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{
+  "name": "langbase-chatbot-example",
```
Comment (msaaddev) [inline examples/ai-drug-assistant/package.json:2]: Please update the project name here

File: examples/ai-drug-assistant/app/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import type { Metadata } from 'next'
+import { Inter } from 'next/font/google'
+import { Toaster } from 'sonner'
+import './globals.css'
+
+const inter = Inter({ subsets: ['latin'] })
+
+export const metadata: Metadata = {
+  title: 'AI Chatbot - Langbase',
+  description: 'Build an AI Chatbot with ⌘ Langbase using any LLM model.',
```
Comment (msaaddev) [inline examples/ai-drug-assistant/app/layout.tsx:12]: Please update name and description here

File: examples/ai-drug-assistant/README.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+3. Download the example project folder from [here][download] or clone the reppository.
+4. `cd` into the project directory and open it in your code editor.
+5. Duplicate the `.env.example` file in this project and rename it to `.env.local`.
+6. Add the following environment variables:
+
+```sh
+# Replace `PIPE_API_KEY` with the copied API key.
+NEXT_LB_PIPE_API_KEY="PIPE_API_KEY"
+
+# Install the dependencies using the following command:
```
Comment (msaaddev) [inline examples/ai-drug-assistant/README.md:40]: The env file and the shell script should be separate code block like

```env.local
# Replace `PIPE_API_KEY` with the copied API key.
NEXT_LB_PIPE_API_KEY="PIPE_API_KEY"
```

Now run the following commands in your terminal:

```sh
# Install the dependencies using the following command:
npm install

# Run the project using the following command:
npm run dev
```

File: examples/ai-drug-assistant/README.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+
+---
+
+## Authors
+
+This project is created by [Langbase][lb] team members, with contributions from:
+
+- Ahmad Awais ([@MrAhmadAwais][xaa]) - Founder & CEO, [Langbase][lb]
+- Ahmad Bilal ([@AhmadBilalDev][xab]) - Founding Engineer, [Langbase][lb]
+- Muhammad-Ali Danish - Software Engineer, [Langbase][lb]
```
Comment (msaaddev) [inline examples/ai-drug-assistant/README.md:60]: Better to only add your name here since you are the author even if you duplicated.

**Review by ahmadbilaldev — CHANGES_REQUESTED** [review]
Update the screenshot. Otherwise LGTM.

**Review by ahmadbilaldev — APPROVED** [review]

**Pull request merged**